index.d.ts

```diff
@@ -11,3 +11,27 @@ export function openInbox({
   cancelLabel?: string;
   removeText?: boolean;
 }): void;
+
+export function openCompose({
```

Comment:
I'm not familiar with writing `index.d.ts` files but I was wondering if both `openCompose` and `openInbox` should be declared as **async** and return a **Promise**

```typescript
export async function openCompose (...): Promise<void>
```

Reply:
@MikeDuister my go-to Typescript guru, could you check the above?